README.md

```diff
@@ -48,6 +48,15 @@ The object has the following schema (validated [here](./lib/index.js) using [Joi
   * `error: Error` - The error that occurred.
   * `reply: Reply` - The hapi.js [reply interface](http://hapijs.com/api#reply-interface).
   > If an error occurs and no function is provided, the request lifecycle continues normally as if there was no token bucket restriction. This is a useful default behavior in case the limitd server goes down.
+* `limitResponseHandler: (limitResult, req, reply) => ()` A function that receives the result of limit checking against limitd, the `request` and the reply interface and is responsible for sending the handling
```

Comment:
I would rename this option to `takeResultHandler`.
Wouldn't it be more generic to handle conformant and non-conformant results? The same way we could `reply.continue()` on non-conformant, it might be useful to reject even for conformant?

If we stay with only non-conformant, then I would rename the option to `nonConformantResultHandler` instead.

---

I would rename `limitResult` to `takeResult` since it's the result object returned from limitd's `take()` method.

---

Maybe we should describe succinctly the default behavior when the option is omitted.

---

> and is responsible for sending the handling how to respond to the customer

This sentence needs better wording. Maybe:
> and is responsible for handling a non-conformant request.

Reply:
`takeResultHandler` seems really tied to limitd and in theory there might be N takes and we will only call the handler one time.
Regarding renaming to take result my main concern is that as final user you should not have to know how limitd works to use patova, so you should not care if limitd has a method called take IMO. 

I will make it more generic more generic to handle the non-limit case too, although in general I resist to make changes I don't feel have a clear use case and fail to see a clear one for the "conformant" case. Maybe not including the headers, or changing some of them I guess? In any case I see some side use cases as the one I pointed out previously, so 👍